src/arraymancer/laser/tensor/datatypes.nim

```diff
@@ -165,43 +170,43 @@ func unsafe_raw_buf*[T: KnownSupportsCopyMem](t: var Tensor[T], aligned: static
   ## and that the data is aligned by LASER_MEM_ALIGN (default 64).
   unsafe_raw_offset_impl(0)
 
-func unsafe_raw_offset*[T: KnownSupportsCopyMem](t: Tensor[T], aligned: static bool = true): RawImmutableView[T] {.inline.} =
+func unsafe_raw_offset*[T: KnownSupportsCopyMem](t: Tensor[T], offset: int = 0, aligned: static bool = true): RawImmutableView[T] {.inline.} =
```

Comment:
What's the use-case of this offset?

Reply:
For consistency with ``toUnsafeView`` but this one is less practical and could be removed if you wish